adminSDK/alertcenter/quickstart/src/main/java/AdminSDKAlertCenterQuickstart.java

```diff
@@ -61,7 +56,7 @@ private static Credentials getCredentials(String delegatedAdminEmail) throws IOE
     GoogleCredentials credentials = ServiceAccountCredentials
         .fromStream(in)
         .createDelegated(delegatedAdminEmail)
-        .createScoped(SCOPES);
+        .createScoped(AlertCenterScopes.all());
```

Comment:
We prefer that the sample uses the minimum scopes required, since it's a best practice. See the Gmail quickstart for an example:

https://github.com/gsuitedevs/java-samples/blob/master/gmail/quickstart/src/main/java/GmailQuickstart.java#L47

Reply:
Done.